packages/repository/package.json

```diff
@@ -15,7 +15,7 @@
     "build:dist10": "lb-tsc es2018",
     "clean": "lb-clean loopback-repository*.tgz dist* package api-docs",
     "pretest": "npm run build",
-    "test": "lb-mocha \"DIST/test/**/*.js\"",
+    "test": "lb-mocha \"DIST/test/**/*.js\" --exclude DIST/test/fixtures/models/bad/*.js",
```

Comment:
What happens if this is not excluded?

Reply:
See https://github.com/strongloop/loopback-next/pull/1618#discussion_r212026384